src/ConsentBanner/ConsentBannerContentDisplayer.tsx

```diff
@@ -0,0 +1,18 @@
+"use client";
+
+import React, { useState, useEffect } from "react";
+import { useGdprStore } from "../useGdprStore";
+import { ConsentBannerContent, ConsentBannerContentProps } from "./ConsentBannerContent";
```

Comment:
```suggestion
import { ConsentBannerContent, type ConsentBannerContentProps } from "./ConsentBannerContent";
```  

Always import types as types

Reply:
Yeah I usually don't even think about it and rely on eslint for that 😅